src/mtool/ui/src/containers/IbofOsOperations/Subsystem/index.js

```diff
@@ -202,6 +204,8 @@ class Subsystem extends Component {
                             }}
                             columns={subsystemTableColumns}
                             icons={icons}
+                            /* eslint-disable react/jsx-props-no-spreading */
```

Comment:
use
// eslint-disable-next-line react/jsx-props-no-spreading
instead of eslint-disable